cmake/packaging.cmake

```diff
@@ -39,6 +39,10 @@ if(WIN32 AND NOT UNIX)
    !include \\\"FileFunc.nsh\\\"\n\
    !include \\\"FileAssociation.nsh\\\"")
 
+  # Retrieve file association
+  get_property(F3D_FILE_ASSOCIATION_NSIS GLOBAL PROPERTY F3D_SUPPORTED_EXTENSIONS)
+  list(TRANSFORM F3D_FILE_ASSOCIATION_NSIS REPLACE "^(.+)$" "'.\\1' '\\1 file'")
```

Comment:
what does this do and why is it necessary ?

Reply:
It replaces the old prepend/append in `f3d_declare_reader` macro:
```cmake
  if(WIN32)
    # used for the NSIS file association during installation (see packaging.cmake)
    set(F3D_FILE_ASSOCIATION_READER "${F3D_READER_EXTENSIONS}")
    list(TRANSFORM F3D_FILE_ASSOCIATION_READER PREPEND "'.")
    list(TRANSFORM F3D_FILE_ASSOCIATION_READER APPEND "' '${F3D_READER_DESCRIPTION}'")
    list(APPEND F3D_FILE_ASSOCIATION_LIST "${F3D_FILE_ASSOCIATION_READER}")
  endif()
```

We do not use `F3D_READER_DESCRIPTION` anymore and I replace it by `[reader_name] file`.  
Related issue: #472